apps/charterafrica/src/payload/blocks/Datasets.js

```diff
@@ -0,0 +1,191 @@
+import { array } from "payload/dist/fields/validations";
+
+const sortOptions = [
+  "metadata_created desc",
+  "metadata_created asc",
+  "name asc",
+  "name desc",
+  "relevance desc",
+  "relevance asc",
+  "metadata_modified desc",
+  "metadata_modified asc",
+  "author asc",
+  "author desc",
+];
+
+const Datasets = {
+  slug: "datasets",
+  fields: [
+    {
+      label: {
+        en: "Sort Options",
+        fr: "Options de tri",
+        pt: "Opções de classificação",
+      },
+      type: "collapsible",
+      fields: [
+        {
+          name: "sortOptions",
+          type: "array",
+          minRows: 1,
+          label: {
+            en: "Options",
+            fr: "Options",
+            pt: "Opções",
+          },
+          fields: [
+            {
+              name: "value",
+              type: "select",
+              options: sortOptions.map((value) => {
+                return {
+                  value,
+                  label: value,
+                };
+              }),
+              unique: true,
+              required: true,
+              validate: (val, options) => {
+                const { data, t } = options || {};
+                if (data?.options?.filter((l) => l.value === val)?.length > 1) {
+                  return t("charterafrica.site:uniqueSortOptions");
+                }
+                return array(val, options);
+              },
+            },
+            {
+              name: "label",
+              type: "text",
+              label: {
+                en: "Label",
+                fr: "Étiquette",
+                pt: "Rótulo",
+              },
+              required: true,
+            },
+          ],
+          admin: {
+            initCollapsed: true,
+            components: {
+              RowLabel: ({ data }) => {
+                return data?.label || data?.value || data?.id;
+              },
+            },
+          },
+        },
+      ],
+    },
+    {
+      label: {
+        en: "Datasets Labels",
```

Comment:
```suggestion
        en: "Search & Filter Labels",
```

We're in Datasets block already, no need to keep mentioning it.